src/index.js

```diff
@@ -30,23 +20,15 @@ module.exports = (/* options */) => ({
   // Fix sourcemaps
   result(stylelintResult, filepath) {
     const lineCorrection = sourceMapsCorrections[filepath]
-    const newWarnings = stylelintResult.warnings.reduce((prevWarnings, warning) => {
-      if (ignoredRules.includes(warning.rule)) return prevWarnings
-      const correctedWarning = Object.assign(warning, {
+    const warnings = stylelintResult.warnings.map(warning =>
+      Object.assign(warning, {
```

Comment:
Hmm for good form I might not mutate `warning` and do `Object.assign({}, warning, {`. Also see my comment on line 32 below.